src/main/java/org/folio/circulation/resources/LoanCollectionResource.java

```diff
@@ -345,4 +347,21 @@ private ItemNotFoundValidator createItemNotFoundValidator(Loan loan) {
         String.format("No item with ID %s could be found", loan.getItemId()),
         ITEM_ID, loan.getItemId()));
   }
+
+  private Result<LoanAndRelatedRecords> refuseWhenItemHasDeclaredLostStatus(
+    Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+
+    return loanAndRelatedRecords
+      .map(LoanAndRelatedRecords::getLoan)
+      .map(Loan::getItem)
+      .next(this::isValidStatus)
```

Comment:
Could this use the `failWhen` method from `Result` instead?

Reply:
Changed